app/client/jsx/Room.jsx

```diff
@@ -83,7 +85,8 @@ class Room extends Component {
             art: <ArtRoom jitsiData={jitsiData} art={roomData.art}></ArtRoom>,
             jitsi: <JitsiVideo jitsiData={jitsiData}></JitsiVideo>,
             iframe: <IFrameRoom jitsiData={jitsiData} iframeOptions={roomData.iframeOptions}></IFrameRoom>,
-            adventure: <Adventure options={roomData} onClick={this.onAdventureClick.bind(this)}></Adventure>
+            adventure: <Adventure options={roomData} onClick={this.onAdventureClick.bind(this)}></Adventure>,
+            library: <LibraryRoom libOptions={roomData.libOptions}></LibraryRoom>
```

Comment:
I would call this `ImageMapRoom` or something more generic, since I can see us using this sort of setup for like, an art gallery.

Reply:
Done. I changed this and similar to ImageMapRoom, and made other changes to be less specific to libraries (e.g. "category" -> "label")